### ISSUE TITLE
Get stats lastUpdate format issue in v0.26.0rc1

### ISSUE THREAD

**Issue by sanders41** [meilisearch/meilisearch#2188]
**Describe the bug**
Prior to v0.26.0rc1 the `lastUpdate` for the stats route was formatted as `2022-02-21T22:31:05.182855908Z` however in v0.26.0rc1 I am getting it as `[2022, 52, 22, 34, 26, 300715005, 0, 0, 0]`. Is this change expected?

**To Reproduce**
`curl -X GET 'http://localhost:7700/stats'`

**Expected behavior**
`lastUpdate` formatted as a date

old format

```py
{'databaseSize': 94208, 'lastUpdate': '2022-02-21T22:31:05.182855908Z', 'indexes': {'indexUID': {'numberOfDocuments': 0, 'isIndexing': False, 'fieldDistribution': {}}, 'indexUID2': {'numberOfDocuments': 0, 'isIn
dexing': False, 'fieldDistribution': {}}}}
```

new format

```py
{'databaseSize': 94208, 'lastUpdate': [2022, 52, 22, 34, 26, 300715005, 0, 0, 0], 'indexes': {'indexUID': {'numberOfDocuments': 0, 'isIndexing': False, 'fieldDistribution': {}}, 'indexUID2': {'numberOfDocuments'
: 0, 'isIndexing': False, 'fieldDistribution': {}}}}
```

**Meilisearch version:** v0.26.0rc1

**Comment by irevoire** [comment]
Bi and it's my fault 🤦‍♂️

**Comment by curquiza** [comment]
Fixed by #2189
